cetmix_tower_yaml/readme/DESCRIPTION.md

```
@@ -1,9 +1,3 @@
-This module implements YAML format data import/export for [Cetmix Tower.](https://cetmix.com/tower)
+This module implements YAML format data import/export for [Cetmix Tower](https://cetmix.com/tower).
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)